lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -113,6 +113,19 @@ magma::ActivateFlowsRequest create_activate_req(
   return req;
 }
 
+magma::ActivateFlowsRequest create_activate_req_for_update_tunnel_ids(
```

Comment:
Do we need `msisdn` here as well? Or that's only for activating actual flows? @pshelar

Reply:
For now, per my understanding, pipelined only needs imsi and ip to find the flow.